app/views/docs/production.phtml

```diff
@@ -1,10 +1,10 @@
-<p>Appwrite default setup is designed to help you get started quickly with using the Appwrite server. To run Appwrite successfully in a production environment, you should follow a few basic concepts and best practices. This document assumes you have some very basic understanding of Docker and Docker Compose command-line tools.</p>
+<p>Appwrite's default setup is designed to help you start building with Appwrite quickly. To succeed with Appwrite in a production environment, you should follow a few basic concepts and best practices. This document assumes you have some basic understanding of Docker and Docker Compose command-line tools.</p>
 
 <h2><a href="/docs/production#encryption" id="encryption">Encryption</a></h2>
 
-<p>By default, the Appwrite setup doesn’t come with a uniquely generated encryption key. This key is used to store your files and sensitive data like webhook passwords or API keys in a safe way. To take advantage of this feature, you must generate a unique key and set it as the value of the <b>_APP_OPENSSL_KEY_V1</b> environment variable.</p>
+<p>Appwrite does not generate a unique encryption key during setup by default. This key encrypts your files and sensitive data like webhook passwords or API keys to keep them secure. To take advantage of this feature, you must generate a unique key and set it as the value of the <code>_APP_OPENSSL_KEY_V1</code> environment variable.</p>
```

Comment:
Is it clear they need to set `_APP_OPENSSL_KEY_V1` right after installation? If you make an account and change the key after that. your password will be gone.

Reply:
Ooooo an important catch!

Reply:
This needs to be added to installation instead, tbh